docs/orleans/toc.yml

```
@@ -10,6 +10,10 @@ items:
         href: benefits.md
       - name: What's new
         href: whats-new-in-orleans.md
+  - name: Get started
+    items:
+      - name: Build your first Orleans app
+        href: quickstarts/build-your-first-orleans-app.md
```

```suggestion
      - name: Build your first Orleans app
        href: quickstarts/build-your-first-orleans-app.md
```

There is already a `Get started` node.

---

That was supposed to be Quickstarts, fixing